en/hosting_guide/server_provisionning.rst

```diff
@@ -0,0 +1,77 @@
+Server provisionning
+====================
+
+Context
+-------
+
+Believe it or not, at some time, your Open Orchestra project will require an external
+server to be deployed on.
+
+If you are doing it at the beginning or the end of the project, by following this documentation
+you will be able to use the same provisionning for your development, integration, staging
+and production server.
+
+Host configuration
+------------------
+
+In your project, we recommand you to create a ``provisionning/hosts`` folder.
+Inside, you can put a host file for each environment, for instance ``prod`` for the production
+environment.
+
+The ``prod`` file will contain the group server with the ssh hosts :
+
+.. code-block::
+
+    [prod]
+    open_orchestra_prod
+
+Environment configuration
+-------------------------
+
+In the ``provisionning/hosts`` folder, you need to create a ``group_vars`` folder which
+will contain all the specific information for each environment. (Still ``prod`` file for
+the production environment)
```

Comment:
the brackets should be rephrased, like "the files should be named as in the hosts folder"